dvc/repo/__init__.py

```diff
@@ -315,6 +321,9 @@ def used_cache(
                 stages = self.stages()
 
             for stage in stages:
+                if stage.is_pkg_import:
+                    continue
+
                 if active and not target and stage.locked:
                     logger.warning(
```

Comment:
not related not this PR: looks like either this message should be moved down the stack or we should add `continue` here.

Reply:
We shouldn't have `continue` here, because we should push/pull/etc cache for the locked stage file itself. We could move this warning to `repo.graph()`, but it would be too specific to use it there. Hence why we put this one here.